### PR TITLE
Added ethereum address faker and test

### PR DESCRIPTION
**Pull request by jackreichelt**

### CODE REVIEW

**Comment by davesag** [comment]
👍 I could definitely use this.

File: lib/index.js [hunk 1]
```diff
@@ -85,6 +85,9 @@ function Faker (opts) {
   var System = require('./system');
   self.system = new System(self);
 
+  var Ethereum = require('./ethereum');
```
Comment (Marak) [inline lib/index.js]: Instead of being a top-level module, `Ethereum` should be sub-classed under `Finance` ( the same as bitcoin currently is )

**Comment by Marak** [comment]
@jackreichelt 

This is good.

Only thing is that ethereum should be inside the `finance` module, instead of it's own top-level faker module.

If you can redo the request I will merge, if not I will try to make the change myself.

Thanks!

**Comment by jackreichelt** [comment]
@Marak I'll update it tomorrow and resubmit this.

**Comment by Marak** [comment]
Resolved with https://github.com/Marak/faker.js/pull/514

**Pull request closed without merging**